cli_tools/diagnostics/main_windows.go

```diff
@@ -274,15 +274,17 @@ func getPlainEventLogs(evts []winEvt, errs chan error) []string {
 
 // getDockerImagesList put docker images list file in logFolder channel and errors in error channel.
 func getDockerImagesList(logs chan logFolder, errs chan error) {
+	dockerImageFilePath := make([]string, 0, 1)
 	dockerPath, err := exec.LookPath("docker")
 	if err != nil {
 		log.Printf("Docker not installed, couldn't get docker images list.\n")
-		return
-	}
-	var commands = []runner{
-		cmd{dockerPath, "image list", dockerImageListFileName, false},
+	} else {
+		var commands = []runner{
+			cmd{dockerPath, "image list", dockerImageListFileName, false},
+		}
+		dockerImageFilePath = runAll(commands, errs)
 	}
-	logs <- logFolder{"Kubernetes", runAll(commands, errs)}
+	logs <- logFolder{"Kubernetes", dockerImageFilePath}
```

Comment:
Is this going to work when gatherLogs [returns the list of paths](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/6eb244551f96a457c908fabc11e2d94a39ed9f33/cli_tools/diagnostics/main.go#L151) and then zipFiles [tries to collect them](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/6eb244551f96a457c908fabc11e2d94a39ed9f33/cli_tools/diagnostics/main.go#L157)?

I looked at zipFiles and I think it will work fine if a logFolder entry has an empty files slice, but please run the tool on a Windows VM without docker to verify this code end-to-end.

Reply:
Yeah, tested on non-docker and docker Windows VM end-to-end, it works